server/flaskr/utils/input_validation.py

```
@@ -21,53 +32,95 @@ def verify_missing_inputs(data: dict, validators: list[InputValidator]) -> list[
+        if value is None:
+            return InputError.RequiredField
+
+        return (
+            InputError.NoError
+            if value in self.enum.__members__
+            else InputError.EnumMemberNotFound
+        )
+
+
+class ListValidator(InputValidator):
+    def __init__(self, field_name, can_be_empty=True):
```

Ruff (0.14.14)

[warning] Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)